packages/wonder-blocks-dropdown/src/components/select-opener.js

```diff
@@ -107,6 +107,9 @@ export default class SelectOpener extends React.Component<SelectOpenerProps> {
 
         const ClickableBehavior = getClickableBehavior(router);
 
+        // Note(TB): Prevents unwanted vertical shift for empty selection
+        const space = "\u00A0";
```

Comment:
**suggestions:** maybe use `tamarab` or `myranae`.  `TB` on its own isn't much to go on.  Also, could you rename the variable to `nbsp` since I would expect `space` to contain `\u0020`.  Lastly, inlining this code (or at least placing the comment down below where `space` is used), I think would make more sense.  As it stands, if I read the comment here, I need to hunt for where `space` is used to understand how its achieving its purpose.

Reply:
**question:** Do you know how a Screen Reader will announce this text?

Reply:
When you read the blank option in the dropdown, it just reads as "text". When the placeholder is the blank spot, it does not read as anything though. I think most of the situations for us are having a blank placeholder.

Updated note reference to tamarab. Moved the code inline and got rid of the variable for space. Moved the comment down to where the nbsp unicode is used.

Reply:
🤔 Maybe for all situations (if possible ofc) it would be nice adding an `aria-label` attribute that could help describing the intention of the dropdown control. No change required, but maybe it would be nice adding some sort of documentation around it.